src/ecmult_impl.h

```diff
@@ -79,21 +81,19 @@ static void secp256k1_ecmult_odd_multiples_table(int n, secp256k1_gej *prej, sec
     d_ge.infinity = 0;
 
     secp256k1_ge_set_gej_zinv(&a_ge, a, &d.z);
-    prej[0].x = a_ge.x;
-    prej[0].y = a_ge.y;
-    prej[0].z = a->z;
-    prej[0].infinity = 0;
+    pre_a[0].x = a->x = a_ge.x;
+    pre_a[0].y = a->y = a_ge.y;
+    pre_a[0].infinity = 0;
 
     zr[0] = d.z;
     for (i = 1; i < n; i++) {
-        secp256k1_gej_add_ge_var(&prej[i], &prej[i-1], &d_ge, &zr[i]);
+        secp256k1_gej_add_ge_var(a, a, &d_ge, &zr[i]);
+        pre_a[i].x = a->x;
+        pre_a[i].y = a->y;
+        pre_a[i].infinity = 0;
     }
 
-    /*
-     * Each point in 'prej' has a z coordinate too small by a factor of 'd.z'. Only
-     * the final point's z coordinate is actually used though, so just update that.
-     */
-    secp256k1_fe_mul(&prej[n-1].z, &prej[n-1].z, &d.z);
```

Comment:
Is it worth keeping at least some version of this comment? For example:

```c
    /*
     * Multiply by d.z to undo the isomorphism that makes 'd' affine.
     */
```

Reply:
Done.